tests/unit/workspace_access/test_clusters.py

```diff
@@ -0,0 +1,41 @@
+from unittest.mock import create_autospec
+
+from databricks.labs.blueprint.installation import Installation
+from databricks.labs.blueprint.tui import MockPrompts
+from databricks.sdk import WorkspaceClient
+from databricks.sdk.service.compute import ClusterDetails
+
+from databricks.labs.ucx.workspace_access.clusters import ClusterAccess
+
+
```

Comment:
Please test the unhappy paths indicated by the `raised` exceptions in the code.

Reply:
Added test case for unhappy path also